text/0000-add-ponycheck-to-stdlib.md

```diff
@@ -0,0 +1,73 @@
+- Feature Name: Add ponycheck to the standard library
+- Start Date: 2022-02-02
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+Add [ponycheck](https://github.com/ponylang/ponycheck) to the Pony standard library.
+
+# Motivation
+
+My motivation for this RFC is twofold:
+
+## I would like to make property based testing more prevalent in the Pony community
+
+I believe that by including a property based testing package in the standard library that more people are likely to discover and use it. I am always wary of adding libraries to the standard library as it can discourage 3rd-party efforts. Writing a property based testing package is a lot of work and so far, ponycheck is the only implementation for Pony. Given the lack of current alternatives and the amount of work it would take to write an alternative, I believe that it is reasonable to assume that we are unlikely to impede the growth of a pony testing ecosystem with this change.
+
+## I would like to be able to use property based testing in the standard library
+
+It isn't currently possible to do any property based testing in the standard library. I believe our test coverage could be improved going forward if property based testing was available as an option.
+
+# Detailed design
+
+From the [ponycheck repository](https://github.com/ponylang/ponycheck):
+
+- Add the `ponycheck` package as a new package in the top-level of the standard library
+- Add the example test application in `examples` to a new folder `examples/ponycheck` in the ponyc repo. The `examples` should follow the README usage already in place for each README in the ponyc repo.
```

Comment:
This addresses my remaining concern. I should be on Sync to say it directly, but just in case I wanted to leave a commenting stating my concern has been addressed.